### PR TITLE
Remove kontrol operations from KiteServer

### LINKED ISSUE #49: Remove kontrol operations from KiteServer class
No need to have a specific method to register a kite to a kontrol instance, but instead we should have dedicated class/method/function for dealing with this case.

### PR DESCRIPTION
**Pull request by usirin**
Fixes #49

### CODE REVIEW

**Comment by usirin** [comment]
This should probably go to `v2` because this is clearly a breaking change. Should we merge this to `master` or `2` branch?

File: src/server/index.js [hunk 1]
```diff
@@ -120,9 +120,9 @@ class KiteServer extends Emitter {
     ws.on('message', rawData => {
       const message = parse(rawData)
       if (this.api.hasMethod(message.method)) {
-        this.handleMessage.call(this, proto, message, ws.kite)
+        this.handleMessage(proto, message, ws.kite)
       } else {
-        if (message.arguments.length == 2) {
+        if (message.arguments.length === 2) {
```
Comment (gokmen) [inline src/server/index.js]: `===` is not necessary here.

**Comment by gokmen** [comment]
I think we can merge this one right now because we are not using `KiteServer` in Koding codebase currently. And with current Kontrol all the existing implementation in `KiteServer` was already broken I believe.

File: src/server/index.js (hunk 1 above)
Comment (usirin) [inline src/server/index.js]: it was because eslint was crying, should we change the rule?

Comment (gokmen) [inline src/server/index.js]: ah ok then.

**Comment by gokmen** [comment]
yep, the only usage that I found is here https://github.com/koding/koding/blob/master/node_modules_koding/bongo/lib/bongo.coffee#L659-L706 which I've disabled it earlier.

**Pull request merged**